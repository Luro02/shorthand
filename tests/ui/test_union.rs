use shorthand::ShortHand;

#[derive(ShortHand)]
#[repr(C)]
union ExampleUnion {
    f1: u32,
    f2: f32,
}

fn main() {}
