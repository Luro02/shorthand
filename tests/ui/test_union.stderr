error: union structs are not supported.
 --> $DIR/test_union.rs:4:1
  |
4 | / #[repr(C)]
5 | | union ExampleUnion {
6 | |     f1: u32,
7 | |     f2: f32,
8 | | }
  | |_^
